virttest/utils_test/qemu.py

```diff
@@ -1528,26 +1559,10 @@ def get_guest_total_mem(cls, os_type, session):
         :param session: VM session.
         :return mem_size: total physical memory in guest.
         """
-        error_context.context("Get physical memory in guest")
-        mem_size = 0.0
-        if os_type == "windows":
-            cmd = 'systeminfo | findstr /C:"Total Physical Memory"'
-            regex = re.compile("Total Physical Memory:\s+(.*)", re.M | re.I)
-        else:
-            cmd = 'cat /proc/meminfo'
-            regex = re.compile("memtotal:\s+(.*)", re.M | re.I)
-        output = session.cmd_output_safe(cmd, timeout=240)
-        try:
-            mem_str = regex.search(output.replace(',', '')).groups()[0]
-            args = ("%sB" % mem_str, "M", 1024)
-            mem_size = float(utils_misc.normalize_data_size(*args))
-            logging.info("total memory reported by os: %.2f MB" % mem_size)
-        except Exception:
-            logging.warn("invalid output from guest: %s" % output)
-        return mem_size
+        return vm.get_current_memory_size()
```

Comment:
@CongSmile  rollback to get_current_memory_size because of:
1. hotplug memory can be detect by bios before reboot.
2. as docstring descript this api report memory detached by OS or bios.